app/assets/javascripts/video-grid.js

```diff
@@ -0,0 +1,355 @@
+var VideoGrid = (function() {
```

Comment:
I think this should be in `/vendor/assets/javascripts` wish it had a bower package.